packages/fast-components-styles-msft/src/badge/index.ts

```diff
@@ -0,0 +1,109 @@
+import {
+    ComponentStyles,
+    ComponentStyleSheet,
+    CSSRules,
+} from "@microsoft/fast-jss-manager";
+import { BadgeClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import {
+    applyLocalizedProperty,
+    Direction,
+    ellipsis,
+    localizeSpacing,
+    toPx,
+} from "@microsoft/fast-jss-utilities";
+import { DesignSystem, withDesignSystemDefaults } from "../design-system/index";
+import { applyTypeRampConfig } from "../utilities/typography";
+import { ensureNormalContrast } from "../utilities/colors";
+import { fontWeight } from "../utilities/fonts";
+
+function smallBadgeStyle(direction: Direction): CSSRules<DesignSystem> {
+    return {
+        padding: localizeSpacing(Direction.ltr)(`0 ${toPx(7)} ${toPx(5)} 0`),
+        height: "13px",
```

Comment:
@nalogiudice would be good to get design feedback on this height being 13px - seems off considering new type ramp values.

Reply:
The final height is not 13px - small comes out to 20px and large to 26px when rendered (as shown in the comp)

Reply:
20px and 26px are correct. If the top and bottom padding have to be different to align the text correctly that's fine, but the type ramp line heights are 16px and 20px respectively.